index.html

```diff
@@ -4,23 +4,42 @@
     <title>Welcome to CYF Chat</title>
   </head>
   <body>
-    <h1>
+    <h1 id ="heading">
       CYF Chat
     </h1>
     <h2>
       Send a message
     </h2>
-    <form action="/messages" method="post">
+    <form id="message-form">
       <p>
-        Name: <input type="text" name="from" placeholder="Your Name" /> <br />
-        Message: <input type="text" name="text" placeholder="The message..." />
+        Name: <input type="text" name="from" placeholder="Your Name" id ="name-id" value =""/> <br />
+        Message: <input type="text" name="text" placeholder="The message..." id ="message-id" value =""/>
```

Comment:
Name `id` attribute could be more meaningfully named in consistent manner. `identifier`-`html control type` The `identifier` could be the `name` attribute of the HTML control. so you may assign `id` attribute for the 1st input control as `from-input` and 2nd input control as `text-input`